.github/workflows/browsers-local.yml

```
@@ -0,0 +1,125 @@
+name: Browsers - local
```

```suggestion
name: Browser Tests
```

It may not be Browserstack, but I wouldn't call it local either.

---

Changed